.maintain/gitlab/check_line_width.sh

```diff
@@ -5,19 +5,20 @@
 set -e
 set -o pipefail
 
-BASE_ORIGIN="origin"
-BASE_BRANCH_NAME="master"
 LINE_WIDTH="120"
 GOOD_LINE_WIDTH="100"
-BASE_BRANCH="${BASE_ORIGIN}/${BASE_BRANCH_NAME}"
 
-git fetch ${BASE_ORIGIN} ${BASE_BRANCH_NAME} --depth 1
-git diff --name-only ${BASE_BRANCH} -- \*.rs | ( while read file
+if [ -z $CI_COMMIT_BEFORE_SHA ]; then
+  echo "No ancestor commit set in \$CI_COMMIT_BEFORE_SHA"
+  exit -1;
+fi
+
+git diff --name-only ${CI_COMMIT_BEFORE_SHA} -- \*.rs | ( while read file
```

Comment:
i assume it shouldn't be compared with the current master `${BASE_BRANCH}` but with that point when that pr forked off from master i.e.
```
git diff --name-only ${BASE_BRANCH}...${CI_COMMIT_SHA} -- \*.rs | ( while read file
```
(three dots)
compare with
https://github.com/paritytech/substrate/blob/master/.maintain/gitlab/check_runtime.sh#L60

Reply:
Yes. And I expect `$CI_COMMIT_BEFORE_SHA` to be that point. `$CI_COMMIT_SHA` is implicit because that is the current `HEAD`. So no need to mention it here.